### DISCUSSION TITLE
Column View Seems Different From the Past

### DISCUSSION THREAD

**Discussion by bjvanhorn** [WinMerge/winmerge#1473] [Q&A] [answered]
Did WinMerge change its gui in some way since I last used it?  I realize that's vague, so I'll explain that the last time I used it, it had two columns, each with a file list or tree for said directory.  Now it appears to show both directories as columns, but the actual file list or tree is somehow combined.  I've attached a screenshot to better illustrate what I'm seeing, and below that is what I more or less used to see.  I'm sure I could adapt to this if necessary, but if I'm just not seeing the option to use the workflow I'm used to, please, for the life of me, help me out.  I really appreciate any help.

![What I see now](https://user-images.githubusercontent.com/12994679/188057113-a098afdd-ea88-47f1-ae31-5a3f1f5b3b8a.png)
![What I used to see](https://user-images.githubusercontent.com/12994679/188057604-1bceffac-7967-45b3-92d8-c00855f26dfb.png)

**Comment by sdottaka** [comment, marked as answer]
WinMerge has never been able to display anything like the second image.
Perhaps you were using a different software

**Reply by bjvanhorn** [reply]
You're right.  I was using Beyond Compare.  Thanks.
